data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -448,7 +448,7 @@ def _load_score_csv_full(
 
     def _load_excel_from_df(
         self, excel_df: pd.DataFrame, excel_path: Path
-    ) -> None:
+    ) -> dict:
```

Comment:
I slightly changed this so that I would not have to redo the i/o BUT am happy to revert it